test/cql-pytest/suite.yaml

```diff
@@ -2,3 +2,9 @@ type: Python
 pool_size: 4
 dirties_cluster:
   - test_native_transport
+extra_scylla_cmdline_options:
+  - '--experimental-features=udf'
+  - '--experimental-features=consistent-topology-changes'
+  - '--experimental-features=broadcast-tables'
```

Comment:
Why did you add this "broadcast tables" here? Do we test them in cql-pytest?

Reply:
Copy pasta, will remove.

Reply:
Removed in v2.